docs/dev_guide/reference/diagnostic_ref/dpct1120.rst

```diff
@@ -0,0 +1,131 @@
+.. _DPCT1120:
+
+DPCT1120
+========
+
+Message
+-------
+
+.. _msg-1120-start:
+
+*<variable name>* is in device memory and passed into device functions as an extra arguments.

```

Comment:
Suggest "as an extra argument" not arguments

Reply:
fixed